cmd/root.go

```diff
@@ -117,6 +133,46 @@ type checkResult struct {
 	Details    []string
 }
 
+type npmsearchresults struct {
+	Objects []struct {
+		Package struct {
+			Name        string    `json:"name"`
+			Scope       string    `json:"scope"`
+			Version     string    `json:"version"`
+			Description string    `json:"description"`
+			Date        time.Time `json:"date"`
+			Links       struct {
+				Npm        string `json:"npm"`
+				Homepage   string `json:"homepage"`
+				Repository string `json:"repository"`
```

Comment:
Since we only need Package.Links.Repository, do we need to define everything else ? What if format slightly changes, will we just error out.

Reply:
Agreed. Removed the other stuff.